{{cookiecutter.repo_name}}/ansible/roles/app/tasks/setup_django_app.yml

```diff
@@ -3,7 +3,7 @@
 - name: Install packages required by the Django app inside virtualenv
   pip: >
     requirements={{ requirements_file }}
-    executable="{{ virtualenv_path }}/bin/pip{{ python_version }}"
+    executable="{{ virtualenv_path }}/bin/pip"
```

Comment:
@stepanovsh does it really works fine now?

earlier it didn't work fine for python 3